### PR TITLE
Made the respective changes for Cisco Meraki Solution Installation Ex…

### PR DESCRIPTION
**Pull request by Tichandr**
…perience for Playbooks, Custom Connector and CreateSolution Tool

-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for Cisco Meraki Playbooks and Custom Connector files

   Reason for Change(s):
   - Improving Customer Installation Experience for Cisco Meraki

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes

_Note: If updating a detection, you must update the version field._

> After the submission has been made, please look at the Validation Checks.

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Tools/Create-Azure-Sentinel-Solution/createSolution.ps1
Comment (elforb) [inline Tools/Create-Azure-Sentinel-Solution/createSolution.ps1:504]: Since this is done at the end of both cases you should be able to just take this logic out of the conditionals and put it below.

**Comment by NikTripathi** [comment]
@anki-narravula Please have a look on playbooks. Thanks

File: Tools/Create-Azure-Sentinel-Solution/createSolution.ps1
Comment (Tichandr) [reply Tools/Create-Azure-Sentinel-Solution/createSolution.ps1]: Yes I have taken care.

**Review by NikTripathi — APPROVED** [review]
Looks good. Can  approve once playbooks are approved.

**Review by anki-narravula — APPROVED** [review]
Looks good. Approved

**Pull request merged**
